Fixes ambiguity of instance-id (#430)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

##### Environment Instance {#sec-comid-instance}

> An `instance-id` is a globally unique value that identifies a Target Environment instance.

Reviewer: ```suggestion
An `instance-id` is a value that identifies a Target Environment instance among other environments of the same class.
```

The identifier is reliably bound to the Target Environment.

> For example, if an X.509 certificate's subject public key is unique for each instance of a target environment, the instance-id might be created from that subject public key.

Reviewer 2: ```suggestion
For example, if an X.509 certificate's subject public key is unique for each instance of a target environment, the `instance-id` might be created from that subject public key.
```

See {{Section 4.1 of -pkix-cert}}.
Alternatively, if the certificate's subject public key is large, the `instance-id` might be a key identifier that is a digest of that public key.
See {{Section 4.2.1.2 of -pkix-cert}}.

> The keyIdentifier is reliably bound to the subjectPublicKey because the identifier is a digest of the key.

Reviewer 2: ```suggestion
The key identifier is reliably bound to the subject public key because the identifier is a digest of the key.
```

##### Environment Instance {#sec-comid-instance}

An `instance-id` is a globally unique value that identifies a Target Environment instance.
The identifier is reliably bound to the Target Environment.

> For example, if the Target Environment is a DICE layer {{DICE.Layer}} the `instance-id` might be an X.509 subjectPublicKey.

Reviewer: ```suggestion
For example, the `instance-id` might be an X.509 subjectPublicKey.
```

Reviewer 2: Suggestion:

> “For example, if an X.509 certificate's subject public key is unique for each instance of a target environment, the `instance-id` might be created from that subject public key."

Reviewer 3: :+1:

> See ({{Section 4.1 of -pkix-cert}}).

Reviewer 2: ```suggestion
See {{Section 4.1 of -pkix-cert}}.
```

The subjectPublicKey is reliably bound to the DICE layer because the key generation is tied to a measurement of the Target Environment.

> Alternatively, if the Target Environment is an X.509 subjectPublicKey the `instance-id` might be a key identifier that is a digest of the public key.

Reviewer 2: ```suggestion
Alternatively, if the Target Environment is an X.509 subjectPublicKey, the `instance-id` might be a key identifier that is a digest of the public key.
```

> The subjectPublicKey is reliably bound to the DICE layer because the key generation is tied to a measurement of the Target Environment.
> Alternatively, if the Target Environment is an X.509 subjectPublicKey the `instance-id` might be a key identifier that is a digest of the public key.

Reviewer: ```suggestion
This is appropriate when the subjectPublicKey used by a device is a unique identifier of the device.
Alternatively, if the subjectPublicKey is large, the `instance-id` might be a key identifier that is a digest of the public key.
```

See {{Section 4.2.1.2 of -pkix-cert}}.

> The keyIdentifier is reliably bound to the subjectPublicKey because the identifier is a digest of the key.

Reviewer 2: By `keyIdentifier` do we mean the `SubjectKeyIdentifier` extension?  Or is it the `KeyIdentifier` ASN.1 type from RFC5280?
